frontend/src/App.js

```diff
@@ -33,6 +34,11 @@ class App extends Component {
 					<NavRoute exact path="/profile" component={Profile} />
 					<NavRoute exact path="/samplemini" component={MiniArticle} />
 					<NavRoute exact path="/account" component={Account} />
+					<NavRoute exact path="/my-articles" component={MyArticles} />
```

Comment:
Should use Route and not NavRoute, since account page does not need the NavBar on top